webapp/src/Entity/Problem.php

```diff
@@ -108,12 +101,23 @@ class Problem extends BaseApiEntity implements
     #[Serializer\Exclude]
     private ?string $problemstatement_type = null;
 
-    #[ORM\Column(options: [
-        'comment' => 'Whether this problem is a multi-pass problem.',
-        'default' => 0,
-    ])]
+    public const TYPE_PASS_FAIL = 1;
```

Comment:
Maybe add a comment for clarity that these are bitfields (even though the powers of 2 kinda reveal it)?

Reply:
Done.